explore/src/main/scala/explore/observationtree/ObsSummaryTable.scala

```diff
@@ -185,21 +187,15 @@ object ObsSummaryTable extends TableHooks:
           .setCell(_.value),
         column(
           RAColumnId,
-          r =>
-            r.asterism
-              .map(_.baseTracking.baseCoordinates.ra) // TODO: baseTrackingAt
-              .orElse(r.targetWithId.map(_.target).flatMap(Target.baseRA.getOption)),
-          r => Target.baseRA.getOption(r.targetWithId.target)
```

Comment:
I don't know if it is possible to default these to the current time since getting the current time is effectful. I can default to base coordinates for now. This will need to change when we support non-sidereals, but at that point we may need to get coordinates from the ODB or somewhere, anyway.